docs/source/blocks/editcomponent.md

```diff
@@ -290,7 +290,7 @@ export const widgets = {
 
 #### Selectable types
 
-If **selectableTypes** is set in _widgetOptions.pattern_options_, widget allows to select only items that matches types defined in _widgetOptions.pattern_options.selectableTypes_.
+If **selectableTypes** is set in `widgetOptions.pattern_options`, then only items whose content type has a name that is defined in `widgetOptions.pattern_options.selectableTypes` will be selectable.
```

Comment:
I think that "selectableTypes" is an inline literal, but if not then please ignore this suggestion. This content is good. Thank you!

```suggestion
If `selectableTypes` is set in `widgetOptions.pattern_options`, then only items whose content type has a name that is defined in `widgetOptions.pattern_options.selectableTypes` will be selectable.
```
```suggestion
If **selectableTypes** is set in `widgetOptions.pattern_options`, then only items whose content type has a name that is defined in `widgetOptions.pattern_options.selectableTypes` will be selectable.
```